contents/product-engineers/what-is-user-stickiness.md

```diff
@@ -0,0 +1,120 @@
+date: 2023-05-09
+title: 'What is user stickiness and how to measure it?'
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+tags:
+  - Product analytics
+  - Product metrics
+  - Product engineers
+  - Growth
+crosspost:
+  - Founders
+  - Blog
+---
+Can’t stop scrolling on TikTok? Endlessly gawping at Instagram? That’s stickiness. Popularized by Meta when it was still known as Facebook, the notion of stickiness remains a crucial measure of how engaging your product or service is regardless of whether you’re a planet-conquering social media app or a B2B SaaS company.
+
+In this guide we’ll explain what user stickiness is, why it’s so important, how to measure it, and give you some tips on how to increase the stickiness of your product.
+
+### What is user stickiness?
+
+Stickiness is a type of insight that lets you see how many people use your product on a regular basis. There are a few different ways to calculate it. DAU/MAU ratio is the most popular, but session frequency, session duration, and individual feature use can also help you understand how sticky your product is.
+
+Customer stickiness is similar but different to customer retention and loyalty. Think of it like this:
+
+* **Customer loyalty** is all about building emotional connections and brand advocacy. Your most loyal customers should love your product. \
+
+* **Customer retention** is a metric that focuses on reducing user churn. You do this by fixing reasons why users might want to leave. \
+
+* **Customer stickiness** is a measure of how well your product keeps users coming back. Achieve that, and you might well have [product-market fit](https://posthog.com/founders/measure-product-market-fit).
+
+All three of these metrics work in tandem: improving customer retention is likely to give you more loyal customers, who are, in turn, more likely to be sticky. Get this right and you’ll have a growing customer base, growing revenue, and growing profits.
+
+### Why is user stickiness important?
+
+However you measure it, stickiness is the lifeblood of a successful product. Acquiring new users can be expensive and time-consuming, so once you’ve onboarded someone you want to keep them engaged and loyal. And, if your product is paid, you want to keep them paying.
```

Comment:
There are a lot of successful products that aren't sticky (one time payments). For a certain type of product, sure. 

Something to think about: B2C vs B2B. People are forced to use Microsoft Teams, so does stickiness matter to them? How should a product like that use stickiness?